### PR TITLE
#1 Feat/login singup redesign final

### LINKED ISSUE #1: 1) Feature: New design for login/signup pages
We’ve received new specs from our designer. Please implement the design for the login and signup pages, and make it mobile responsive. The design spec and needed assets are in the provided zip folder

### PR DESCRIPTION
**Pull request by camba95**
Closes #1 

Changes include:
- mobile design
- UI improvements
- Email as credential instead of username
Screen Shot 2021-04-22 at 8 11 10 PM
Screen Shot 2021-04-22 at 8 11 17 PM
Screen Shot 2021-04-22 at 8 11 28 PM
Screen Shot 2021-04-22 at 8 11 39 PM

### CODE REVIEW

File: client/src/components/SideBanner/SideBanner.js [hunk 1]
```diff
@@ -30,6 +30,11 @@ const useStyles = makeStyles({
 const SideBanner = (props) => {
   const classes = useStyles();
   const { children } = props;
+
+  const smallScreen = useMediaQuery(theme => theme.breakpoints.down('sm'));
+  if (smallScreen) {
+    return children;
+  }
```
Comment (WhoaNellie) [inline client/src/components/SideBanner/SideBanner.js]: You might be able to use [Hidden](https://material-ui.com/components/hidden/#hidden) in the parent to remove the side banner on small screens

File: client/src/pages/Login/Login.js [hunk 2]
```diff
@@ -22,34 +50,43 @@ const Login = (props) => {
...
-          <Box m={2}>
-            <Typography>Don't have an account?</Typography>
-          </Box>
+      <Box className={classes.root}>
+        <Grid container justify="flex-end" alignItems="center">
+          {!smallScreen && (
+            <Box m={2}>
+              <Typography>Don't have an account?</Typography>
+            </Box>
+          )}
```
Comment (WhoaNellie) [inline client/src/pages/Login/Login.js]: If you decide to use Hidden you could also use it here

File: client/src/pages/Signup/Signup.js [hunk 3]
```diff
@@ -7,49 +7,78 @@ import {
...
+    width: "60%",
+    margin: "0 auto",
+    marginTop: "10rem",
+    [theme.breakpoints.down('sm')]: {
+      marginTop: "2rem",
+    },
+  }
+}));
+
 const Signup = (props) => {
```
Comment (WhoaNellie) [inline client/src/pages/Signup/Signup.js:40]: This component seems like it could be combined with Login as they're very similar

**Review by WhoaNellie — COMMENTED** [review]
Suggested an MUI component to optionally render based on screen size, could also eliminate the nees for the `renderFormTitle `, `renderLoginButton` and `renderCreateAccountButton` functions.

For PR quality: It's helpful to add screenshots of UI changes so reviewers can see what the styling looks like without having to run the code!

**Review by bonnieli — APPROVED** [review]

**Pull request merged**